enso_driver.py

```diff
@@ -0,0 +1,37 @@
+import os
+from acme_diags.parameter.core_parameter import CoreParameter
+from acme_diags.parameter.enso_diags_parameter import EnsoDiagsParameter
+from acme_diags.run import runner
+
+# Generic parameters. See enso.cfg for specific parameters.
+# Run with `python enso_driver.py -d enso.cfg`.
+param = CoreParameter()
+machine_path_prefix = '/global/project/projectdirs/acme/acme_diags'
+param.reference_data_path = os.path.join(machine_path_prefix, 'obs_for_e3sm_diags/time-series')
+param.test_data_path = os.path.join(machine_path_prefix, 'test_model_data_for_acme_diags/time-series/E3SM_v1')
+html_prefix = '/global/project/projectdirs/acme/www/forsyth'
+param.results_dir = os.path.join(html_prefix, 'enso_diags_12_23')
+param.output_file = os.path.join(param.results_dir, 'enso_diags/variable/prect-ann/plot')
```

Comment:
Is this what we want `output_file` to be? It seems like this should be auto-generated.

Reply:
Also, `enso_diags/variable` and `enso_diags/variable/prect-ann/` need to be created manually. It seems like this should be automatically created if it doesn't exist. If `case_id` is set to `variable` then only `enso_diags/variable/prect-ann/` needs to be created.

Reply:
> Is this what we want `output_file` to be? It seems like this should be auto-generated.

`output_file` depends on the diag being created and is based on the var, region, season, or something similar. Please see other examples in the `*_driver.py` files.

---

Is this `enso_driver.py` a "driver" that for plotset? Meaning, do users following [a guide like this](https://e3sm-project.github.io/e3sm_diags/docs/html/quickguides/quick-guide-compy.html#running-the-annual-mean-latitude-longitude-contour-set) call it via:

```python
runner.sets_to_run = ['enso']
```

Currently, this seems like something else?

Reply:
Ok, I'll work on auto-generating `output_file`.
No, the driver is `enso_diags_driver.py` - this file should be renamed something like `run_enso_diags.py`.